app/forms/candidate_interface/restructured_work_history/work_history_break_form.rb

```diff
@@ -0,0 +1,64 @@
+module CandidateInterface
+  module RestructuredWorkHistory
+    class WorkHistoryBreakForm
```

Comment:
this doesn't have any tests at the moment. Can we grab the tests for it form here please rms/candidate_interface/work_history_break_form_spec.rb